lib/flutter_camera_ml_vision.dart

```diff
@@ -158,8 +158,11 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>> {
 
       return;
     }
-    _cameraController = CameraController(description,
-        Platform.isIOS ? ResolutionPreset.low : ResolutionPreset.medium);
+    _cameraController = CameraController(
+      description,
+      Platform.isIOS ? ResolutionPreset.low : ResolutionPreset.medium,
```

Comment:
not the purpose of this PR but we might want to let users put a resolution their self (and if not set put what we have here)

Reply:
That's right could you open a new issues for that ?
we can pass the two in 2.2.0